src/theme/mod.rs

```diff
@@ -361,7 +362,8 @@ impl render::SizeColours for Theme {
 #[cfg(unix)]
 impl render::UserColours for Theme {
     fn you(&self)           -> Style { self.ui.users.user_you }
-    fn someone_else(&self)  -> Style { self.ui.users.user_someone_else }
+    fn someone_else(&self)  -> Style { self.ui.users.user_root }
```

Comment:
Could we also refactor so this matches as well?
```suggestion
    fn other(&self)  -> Style { self.ui.users.user_root }
```